workflows-scripts/data-export/jmeter-supported-data/50-instance-record-ids.csv

```diff
@@ -0,0 +1,40 @@
+"83ecfb29-dddc-4891-83c0-6d87633d1a54"
```

Comment:
There are 40 Ids in this file instead of 50. Does it makes a difference in performance if it was 40 and not 50 IDs? Same questions as in the 10-instance-record-ids.csv

Reply:
Yes, the number of IDs make a difference in performance.
These IDs were provided to us by PO and they were from bugfest env.
These are slow ones from bugfest env.

Reply:
Will substitute them with another set of IDs yield the same performance or better, or worse?

Reply:
That's a good question. I think it depends on the query used to filter out the result. I will need to reach out to PO to understand why they typed that query which generated these IDs.

Reply:
Could you grab a list of 10 random instance IDs and see if you could replicate the same performance slowness when running the JMeter script in bugfest? I don't want to be tightly coupled with a particular set of records. Ideally the performance problem is agnostic of the IDs,  unless the IDs has hundreds of associated IDs from other tables which makes a JOIN slow, for example.  

If any set of 10 IDs reproduce the perf behavior, then it's good to note that, too, in the README, so that if some IDs happened to be gone off the perf database then we could easily replace them with other IDs.

Reply:
Yes, I can but either way, it depends on what is the query.
> I don't want to be tightly coupled with a particular set of records.
IDs are tightly coupled to query. Therefore, if we want to really decouple it, then the query should be randomly generated. That means we should come up with 10 random queries that are most frequently used.

Reply:
As far as I know, `source= marc` is the most common so maybe I can start with it.

Reply:
My point is just to see if we can get this same problem with any 10 IDs.  You could go to the db and grab any 10 IDs from there and see if the same perf issue is observed. If so then we can add a note saying that any IDs could work, not these chosen ones.  We don't need to come up with random queries or anything more elaborate.